pytype/stubs/builtins/3/builtins.pytd

```diff
@@ -3,7 +3,7 @@
 
 
 from typing import (
-    AbstractSet, Any, AnyStr, AsyncGenerator, Awaitable, BinaryIO, Callable,
+    AbstractSet, Annotated, Any, AnyStr, AsyncGenerator, Awaitable, BinaryIO, Callable,
```

Comment:
Annotated doesn't need to be imported here. (The imports here are for typing members that are needed to type-annotate builtins.)

Reply:
Thanks for explaining. I've removed the import and added a stub instead.